### ISSUE TITLE
Saving parts imported from SBH generates confusing sbol:sequence URL

### ISSUE THREAD

**Issue by ImperialiGEM2020** [SynBioDex/SBOLCanvas#107]
Importing a part from iGEM collection in the server synbiohub.org results in the generation of a URL (that does not lead anywhere) within the sbol:ComponentDefinition linking to the sequence of the part, like so for component with displayId BBa_R0052

```<sbol:sequence rdf:resource="https://sbolcanvas.org/BBa_R0052Sequence"/>```

The sequence is then retrieved within the sbol:Sequence block with displayId BBa_R0052Sequence as sbol:elements and the correct link in sbol:encoding :

```<sbol:Sequence rdf:about="https://sbolcanvas.org/BBa_R0052Sequence">
    <sbol:persistentIdentity rdf:resource="https://sbolcanvas.org/BBa_R0052Sequence"/>
    <sbol:displayId>BBa_R0052Sequence</sbol:displayId>
    <sbol:elements>ttgacaaacaagatacattgtatgaaaatacaagaaagtttgttga</sbol:elements>
    <sbol:encoding rdf:resource="http://www.chem.qmul.ac.uk/iubmb/misc/naseq.html"/>
  </sbol:Sequence>
```

Is this expected and can SBOL functions then correctly extract the sequence?

**Comment by cjmyers** [comment]
Good catch.  This is also a problem that we have noticed and are now working on, see #99.

**Comment by cjmyers** [comment]
Closed by PR #108
